src/main/java/com/example/LooseCouplingDemo/service_implementation/MessagesServiceInterfaceImpl.java

```diff
@@ -0,0 +1,40 @@
+package com.example.LooseCouplingDemo.service_implementation;
```

Comment:
Why do we have these 2 classes?
MessagesServiceInterfaceImpl
MessageService

Reply:
i used this for testing purpose.